pkg/acs/calacs/lib/prinfo.c

```diff
@@ -292,15 +292,15 @@ int maxch     i: allocated length of outstr (including EOS)
    to lower case, and pad with blanks.
 */
 
-void Upper1 (char *instr, char *outstr, int maxch) {
+void Upper1 (const char *instr, char *outstr, int maxch) {
 
 /* arguments:
 char *instr   i: input string
 char *outstr  o: string with only the first letter capitalized
 int maxch     i: allocated length of outstr (including EOS)
 */
 
-	int i;
+	int i = 1;
```

Comment:
I'm with you it would be much better to do
```
if (!instr || instr == '\0')
{
    outstr = '\0';
    return;
}
```

but the code looks like it pads and then even adds another null char at the end of the pad.